kubetools/config.py

```diff
@@ -99,6 +99,7 @@ def load_kubetools_config(
         ).format(' ({0})'.format(app_name) if app_name else ''))
 
     config = yaml.safe_load(config)
+    config['_filename'] = filename
```

Comment:
Should this not be popped off for k8s, or are underscore keys ignored?

Reply:
No top level keys map to K8s objects luckily, that's all nested under `deployments`, `dependencies` & `upgrades`! Verified by the new tests (https://github.com/EDITD/kubetools/tree/master/tests/configs/basic_app) :)